hash/msx1_cart.xml

```diff
@@ -26313,8 +26221,7 @@ legacy FM implementations cannot find it.
 	<software name="trail2">
 		<description>Trail 2</description>
 		<year>2012</year>
-		<publisher>&lt;doujin&gt;</publisher>
-		<info name="developer" value="Kotai"/>
+		<publisher>Kotai</publisher>
```

Comment:
Gee, I can’t even tell how this should be capitalised – the guy uses “kotai”, “KOTAI” and “Kotai” in different places (it’s from “Konami & Taito Fan”).